
running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 

Building dependencies ... ok

FAILED TEST: tests/actual_tests/bad_pattern.rs
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests/bad_pattern.rs" "--edition" "2021"

error: actual output differed from expected
Execute `DO NOT BLESS. These are meant to fail` to update `tests/actual_tests/bad_pattern.stderr` to the actual output
--- tests/actual_tests/bad_pattern.stderr
+++ <stderr output>
 error[E0308]: mismatched types
  --> tests/actual_tests/bad_pattern.rs:4:9
... 10 lines skipped ...
   |        ^^^
 
-error: aborting due to previous error
+error: aborting due to 1 previous error
 
 For more information about this error, try `rustc --explain E0308`.


error: `miesmätsched types` not found in diagnostics on line 4
 --> tests/actual_tests/bad_pattern.rs:5:17
  |
5 |     //~^ ERROR: miesmätsched types
  |                 ^^^^^^^^^^^^^^^^^^ expected because of this pattern
  |

error: there were 1 unmatched diagnostics
 --> tests/actual_tests/bad_pattern.rs:4:9
  |
4 |     add("42", 3);
  |         ^^^^ Error[E0308]: mismatched types
  |

full stderr:
error[E0308]: mismatched types
 --> tests/actual_tests/bad_pattern.rs:4:9
  |
4 |     add("42", 3);
  |     --- ^^^^ expected `usize`, found `&str`
  |     |
  |     arguments to this function are incorrect
  |
note: function defined here
 --> $DIR/tests/integrations/basic-fail/src/lib.rs:LL:CC
  |
1 | pub fn add(left: usize, right: usize) -> usize {
  |        ^^^

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.

full stdout:



FAILED TEST: tests/actual_tests/executable.rs
command: "$CMD"

error: actual output differed from expected
Execute `DO NOT BLESS. These are meant to fail` to update `tests/actual_tests/executable.run.stdout` to the actual output
--- tests/actual_tests/executable.run.stdout
+++ <stdout output>
-69
+42


full stderr:

full stdout:



FAILED TEST: tests/actual_tests/executable_compile_err.rs
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests/executable_compile_err.rs" "--edition" "2021"

error: pass test got exit status: 1, but expected 0

error: actual output differed from expected
Execute `DO NOT BLESS. These are meant to fail` to update `tests/actual_tests/executable_compile_err.stderr` to the actual output
--- tests/actual_tests/executable_compile_err.stderr
+++ <stderr output>
+error: this file contains an unclosed delimiter
+ --> tests/actual_tests/executable_compile_err.rs:4:2
+  |
+3 | fn main() {
+  |           - unclosed delimiter
+4 |
+  |  ^
+
+error: aborting due to 1 previous error
+


error: there were 1 unmatched diagnostics
 --> tests/actual_tests/executable_compile_err.rs:4:1
  |
4 | 
  | ^ Error: this file contains an unclosed delimiter
  |

full stderr:
error: this file contains an unclosed delimiter
 --> tests/actual_tests/executable_compile_err.rs:4:2
  |
3 | fn main() {
  |           - unclosed delimiter
4 |
  |  ^

error: aborting due to 1 previous error


full stdout:



FAILED TEST: tests/actual_tests/exit_code_fail.rs
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests/exit_code_fail.rs" "--edition" "2021"

error: fail test got exit status: 0, but expected 1

error: no error patterns found in fail test

full stderr:

full stdout:



FAILED TEST: tests/actual_tests/filters.rs
command: "parse comments"

error: `x86_64` is not a valid condition, expected `on-host`, /[0-9]+bit/, /host-.*/, or /target-.*/
 --> tests/actual_tests/filters.rs:1:9
  |
1 | //@only-x86_64
  |         ^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests/foomp.rs
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests/foomp.rs" "--edition" "2021"

error: actual output differed from expected
Execute `DO NOT BLESS. These are meant to fail` to update `tests/actual_tests/foomp.stderr` to the actual output
--- tests/actual_tests/foomp.stderr
+++ <stderr output>
 error[E0308]: mismatched types
  --> tests/actual_tests/foomp.rs:4:9
   |
 4 |     add("42", 3);
   |     --- ^^^^ expected `usize`, found `&str`
+  |     |
+  |     arguments to this function are incorrect
   |
 note: function defined here
- --> $DIR/tests/integrations/basic/src/lib.rs:LL:CC
+ --> $DIR/tests/integrations/basic-fail/src/lib.rs:LL:CC
   |
 1 | pub fn add(left: usize, right: usize) -> usize {
-  |        ^^^ some expected text that isn't in the actual message
+  |        ^^^
 
-error: aborting doo to previous error
+error: aborting due to 1 previous error
 
 For more information about this error, try `rustc --explain E0308`.


full stderr:
error[E0308]: mismatched types
 --> tests/actual_tests/foomp.rs:4:9
  |
4 |     add("42", 3);
  |     --- ^^^^ expected `usize`, found `&str`
  |     |
  |     arguments to this function are incorrect
  |
note: function defined here
 --> $DIR/tests/integrations/basic-fail/src/lib.rs:LL:CC
  |
1 | pub fn add(left: usize, right: usize) -> usize {
  |        ^^^

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.

full stdout:



FAILED TEST: tests/actual_tests/foomp2.rs
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests/foomp2.rs" "--edition" "2021"

error: actual output differed from expected
Execute `DO NOT BLESS. These are meant to fail` to update `tests/actual_tests/foomp2.stderr` to the actual output
--- tests/actual_tests/foomp2.stderr
+++ <stderr output>
 error[E0308]: mismatched types
  --> tests/actual_tests/foomp2.rs:4:9
... 10 lines skipped ...
   |        ^^^
 
-error: aborting due to previous error
+error: aborting due to 1 previous error
 
 For more information about this error, try `rustc --explain E0308`.


full stderr:
error[E0308]: mismatched types
 --> tests/actual_tests/foomp2.rs:4:9
  |
4 |     add("42", 3);
  |     --- ^^^^ expected `usize`, found `&str`
  |     |
  |     arguments to this function are incorrect
  |
note: function defined here
 --> $DIR/tests/integrations/basic-fail/src/lib.rs:LL:CC
  |
1 | pub fn add(left: usize, right: usize) -> usize {
  |        ^^^

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.

full stdout:



FAILED TEST: tests/actual_tests/pattern_too_many_arrow.rs
command: "parse comments"

error: //~^ pattern is trying to refer to 7 lines above, but there are only 2 lines above
 --> tests/actual_tests/pattern_too_many_arrow.rs:3:8
  |
3 |     //~^^^^^^^ ERROR: mismatched types
  |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests/pattern_too_many_arrow_above.rs
command: "parse comments"

error: //~v pattern is trying to refer to line 8, but the file only has 6 lines
 --> tests/actual_tests/pattern_too_many_arrow_above.rs:2:22
  |
2 |     //~vvvvvv ERROR: mismatched types
  |                      ^^^^^^^^^^^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests/rustc_ice.rs
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests/rustc_ice.rs" "-Ztreat-err-as-bug" "--edition" "2021"

error: fail test got exit status: 101, but expected 1

error: `mismatched types` not found in diagnostics on line 8
 --> tests/actual_tests/rustc_ice.rs:9:17
  |
9 |     //~^ ERROR: mismatched types
  |                 ^^^^^^^^^^^^^^^^ expected because of this pattern
  |

error: there were 1 unmatched diagnostics
 --> tests/actual_tests/rustc_ice.rs:8:5
  |
8 |     add("42", 3);
  |     ^^^^^^^^^^^^ Ice: no errors reported for args
  |

full stderr:
error: internal compiler error: no errors reported for args
 --> tests/actual_tests/rustc_ice.rs:8:5
  |
8 |     add("42", 3);
  |     ^^^^^^^^^^^^

thread 'rustc' panicked at compiler/rustc_errors/src/lib.rs:
aborting due to `-Z treat-err-as-bug=1`
stack backtrace:

error: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust/issues/new?labels=C-bug%2C+I-ICE%2C+T-compiler&template=ice.md



note: compiler flags: -Z treat-err-as-bug

query stack during panic:
#0 [typeck] type-checking `main`
#1 [analysis] running analysis passes on this crate
end of query stack

full stdout:


FAILURES:
    tests/actual_tests/bad_pattern.rs
    tests/actual_tests/executable.rs
    tests/actual_tests/executable_compile_err.rs
    tests/actual_tests/exit_code_fail.rs
    tests/actual_tests/filters.rs
    tests/actual_tests/foomp.rs
    tests/actual_tests/foomp2.rs
    tests/actual_tests/pattern_too_many_arrow.rs
    tests/actual_tests/pattern_too_many_arrow_above.rs
    tests/actual_tests/rustc_ice.rs

test result: FAIL. 10 failed;

Building dependencies ... ok
Building aux file tests/actual_tests_bless/auxiliary/the_proc_macro.rs ... ok
Building aux file tests/actual_tests_bless/auxiliary/foomp.rs ... ok
Building aux file tests/actual_tests_bless/auxiliary/nested.rs ... ok

FAILED TEST: tests/actual_tests_bless/aux_build_not_found.rs
command: "$CMD"

full stderr:
No such file or directory
full stdout:



FAILED TEST: tests/actual_tests_bless/aux_proc_macro_misuse.rs
command: "parse comments"

error: proc macros are now auto-detected, you can remove the `:proc-macro` after the file name
 --> tests/actual_tests_bless/aux_proc_macro_misuse.rs:1:32
  |
1 | //@aux-build:the_proc_macro.rs:proc-macro
  |                                ^^^^^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/aux_proc_macro_no_main.rs
command: "rustc" "--error-format=json" "--crate-type=lib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/aux_proc_macro_no_main.rs" "--edition" "2021" "--extern" "the_proc_macro=$DIR/tests/integrations/basic-fail/../../../target/$TMP/tests/actual_tests_bless/auxiliary/libthe_proc_macro.so" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/tests/actual_tests_bless/auxiliary"

error: there were 1 unmatched diagnostics
 --> tests/actual_tests_bless/aux_proc_macro_no_main.rs:7:8
  |
7 | thing!(cake);
  |        ^^^^ Error: expected one of `!` or `::`, found `<eof>`
  |

error: no error patterns found in fail test

full stderr:
error: expected one of `!` or `::`, found `<eof>`
 --> tests/actual_tests_bless/aux_proc_macro_no_main.rs:7:8
  |
7 | thing!(cake);
  |        ^^^^ expected one of `!` or `::`

error: aborting due to 1 previous error


full stdout:



FAILED TEST: tests/actual_tests_bless/compile_flags_quotes.rs
command: "parse comments"

error: `compile-flag` is not a command known to `ui_test`, did you mean `compile-flags`?
 --> tests/actual_tests_bless/compile_flags_quotes.rs:1:4
  |
1 | //@compile-flag: cake
  |    ^^^^^^^^^^^^
  |

error: `-Z "cheese is good` contains an unclosed quotation mark
 --> tests/actual_tests_bless/compile_flags_quotes.rs:3:19
  |
3 | //@compile-flags: -Z "cheese is good
  |                   ^^^^^^^^^^^^^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/compiletest-rs-command.rs
command: "parse comments"

error: a compiletest-rs style comment was detected.
Please use text that could not also be interpreted as a command,
and prefix all actual commands with `//@`
 --> tests/actual_tests_bless/compiletest-rs-command.rs:1:3
  |
1 | //aux-build:foomp.rs
  |   ^^^^^^^^^^^^^^^^^^
  |

error: a compiletest-rs style comment was detected.
Please use text that could not also be interpreted as a command,
and prefix all actual commands with `//@`
 --> tests/actual_tests_bless/compiletest-rs-command.rs:2:4
  |
2 | // aux-build:asf.rs
  |    ^^^^^^^^^^^^^^^^
  |

error: comment looks suspiciously like a test suite command: `@aux-build:asldkfjasldfj.rs`
All `//@` test suite commands must be at the start of the line.
The `//` must be directly followed by `@` or `~`.
 --> tests/actual_tests_bless/compiletest-rs-command.rs:4:4
  |
4 | // @aux-build:asldkfjasldfj.rs
  |    ^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/failing_executable.rs
command: "$CMD"

error: run(0) test got exit status: 101, but expected 0

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/no_main.rs
command: "rustc" "--error-format=json" "--crate-type=lib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/no_main.rs" "--edition" "2021"

error: fail test got exit status: 0, but expected 1

error: no error patterns found in fail test

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/no_main_manual.rs
command: "rustc" "--error-format=json" "--crate-type=lib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/no_main_manual.rs" "--crate-type=bin" "--edition" "2021"

error: there were 1 unmatched diagnostics that occurred outside the testfile and had no pattern
    Error: cannot mix `bin` crate type with others

error: there were 1 unmatched diagnostics
 --> tests/actual_tests_bless/no_main_manual.rs:3:16
  |
3 | pub fn foo() {}
  |                ^ Error[E0601]: `main` function not found in crate `no_main_manual`
  |

error: no error patterns found in fail test

full stderr:
error: cannot mix `bin` crate type with others

error[E0601]: `main` function not found in crate `no_main_manual`
 --> tests/actual_tests_bless/no_main_manual.rs:3:16
  |
3 | pub fn foo() {}
  |                ^ consider adding a `main` function to `tests/actual_tests_bless/no_main_manual.rs`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0601`.

full stdout:



FAILED TEST: tests/actual_tests_bless/no_test.rs
command: "rustc" "--error-format=json" "--test" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/no_test.rs" "--edition" "2021"

error: fail test got exit status: 0, but expected 1

error: no error patterns found in fail test

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/non_top_level_configs.rs
command: "parse comments"

error: comment looks suspiciously like a test suite command: `@check-pass`
All `//@` test suite commands must be at the start of the line.
The `//` must be directly followed by `@` or `~`.
 --> tests/actual_tests_bless/non_top_level_configs.rs:1:5
  |
1 | ////@check-pass
  |     ^
  |

error: comment looks suspiciously like a test suite command: `@ignore-target-cheesecake`
All `//@` test suite commands must be at the start of the line.
The `//` must be directly followed by `@` or `~`.
 --> tests/actual_tests_bless/non_top_level_configs.rs:4:7
  |
4 |     //@ignore-target-cheesecake
  |       ^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/pass_with_annotation.rs
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/pass_with_annotation.rs" "--edition" "2021"

error: `the cake is a lie` not found in diagnostics on line 3
 --> tests/actual_tests_bless/pass_with_annotation.rs:3:12
  |
3 | //~ ERROR: the cake is a lie
  |            ^^^^^^^^^^^^^^^^^ expected because of this pattern
  |

error: error pattern found in pass test
 --> tests/actual_tests_bless/pass_with_annotation.rs:3:12
  |
3 | //~ ERROR: the cake is a lie
  |            ^^^^^^^^^^^^^^^^^ expected because of this annotation
  |
 ::: tests/actual_tests_bless/pass_with_annotation.rs:1:4
  |
1 | //@check-pass
  |    ^^^^^^^^^^ expected because of this mode change
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/revised_revision.rs
command: "parse comments"

error: revisions cannot be declared under a revision
 --> tests/actual_tests_bless/revised_revision.rs:2:5
  |
2 | //@[foo] revisions: foo bar
  |     ^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/revisioned_executable.rs (revision `panic`)
command: "$CMD"

error: run(101) test got exit status: 0, but expected 101

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/revisioned_executable_panic.rs (revision `run`)
command: "$CMD"

error: run(0) test got exit status: 101, but expected 0

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/revisions_bad.rs (revision `bar`)
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/revisions_bad.rs" "--cfg=bar" "--edition" "2021"

error: ``main` function not found in crate `revisions_bad`` not found in diagnostics outside the testfile
 --> tests/actual_tests_bless/revisions_bad.rs:4:31
  |
4 | //@[bar] error-in-other-file: `main` function not found in crate `revisions_bad`
  |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected because of this pattern
  |

error: there were 1 unmatched diagnostics
  --> tests/actual_tests_bless/revisions_bad.rs:10:2
   |
10 | }
   |  ^ Error[E0601]: `main` function not found in crate `revisions_bad`
   |

full stderr:
error[E0601]: `main` function not found in crate `revisions_bad`
  --> tests/actual_tests_bless/revisions_bad.rs:10:2
   |
10 | }
   |  ^ consider adding a `main` function to `tests/actual_tests_bless/revisions_bad.rs`

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0601`.

full stdout:



FAILED TEST: tests/actual_tests_bless/rustfix-fail-revisions.rs (revision `a`)
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/rustfix-fail-revisions.a.fixed" "--cfg=a" "--edition" "2021" "--crate-name" "rustfix_fail_revisions"

error: rustfix failed with exit status: 1

full stderr:
error[E0382]: use of moved value: `x`
  --> tests/actual_tests_bless/rustfix-fail-revisions.a.fixed:12:5
   |
5  |     let x = match 0 {
   |         - move occurs because `x` has type `String`, which does not implement the `Copy` trait
...
11 |     x;
   |     - value moved here
12 |     x;
   |     ^ value used here after move
   |
help: consider cloning the value if the performance cost is acceptable
   |
11 |     x.clone();
   |      ++++++++

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0382`.

full stdout:



FAILED TEST: tests/actual_tests_bless/rustfix-fail-revisions.rs (revision `b`)
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/rustfix-fail-revisions.b.fixed" "--cfg=b" "--edition" "2021" "--crate-name" "rustfix_fail_revisions"

error: rustfix failed with exit status: 1

full stderr:
error[E0382]: use of moved value: `x`
  --> tests/actual_tests_bless/rustfix-fail-revisions.b.fixed:12:5
   |
5  |     let x = match 0 {
   |         - move occurs because `x` has type `String`, which does not implement the `Copy` trait
...
11 |     x;
   |     - value moved here
12 |     x;
   |     ^ value used here after move
   |
help: consider cloning the value if the performance cost is acceptable
   |
11 |     x.clone();
   |      ++++++++

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0382`.

full stdout:



FAILED TEST: tests/actual_tests_bless/rustfix-fail.rs
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/rustfix-fail.fixed" "--edition" "2021" "--crate-name" "rustfix_fail"

error: rustfix failed with exit status: 1

full stderr:
error[E0382]: use of moved value: `x`
  --> tests/actual_tests_bless/rustfix-fail.fixed:11:5
   |
4  |     let x = match 0 {
   |         - move occurs because `x` has type `String`, which does not implement the `Copy` trait
...
10 |     x;
   |     - value moved here
11 |     x;
   |     ^ value used here after move
   |
help: consider cloning the value if the performance cost is acceptable
   |
10 |     x.clone();
   |      ++++++++

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0382`.

full stdout:



FAILED TEST: tests/actual_tests_bless/unknown_revision.rs
command: "parse comments"

error: there are no revisions in this test
 --> tests/actual_tests_bless/unknown_revision.rs:3:9
  |
3 |     //~[cake, lie] ERROR: kawoom
  |         ^^^^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/unknown_revision2.rs
command: "parse comments"

error: the revision `cake` is not known
 --> tests/actual_tests_bless/unknown_revision2.rs:5:9
  |
5 |     //~[cake, lie] ERROR: kawoom
  |         ^^^^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests_bless/wrong_diagnostic_code.rs
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless/wrong_diagnostic_code.rs" "--edition" "2021"

error: diagnostic code `should_be_dead_code` not found on line 3
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:4:10
  |
4 |     //~^ should_be_dead_code
  |          ^^^^^^^^^^^^^^^^^^^ expected because of this pattern
  |

error: there were 1 unmatched diagnostics
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:3:4
  |
3 | fn foo() -> i32 {
  |    ^^^ Error[dead_code]: function `foo` is never used
  |

full stderr:
error: unreachable expression
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:6:5
  |
5 |     panic!();
  |     -------- any code following this expression is unreachable
6 |     0
  |     ^ unreachable expression
  |
note: the lint level is defined here
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:1:20
  |
1 | #![deny(dead_code, unreachable_code)]
  |                    ^^^^^^^^^^^^^^^^

error: function `foo` is never used
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:3:4
  |
3 | fn foo() -> i32 {
  |    ^^^
  |
note: the lint level is defined here
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:1:9
  |
1 | #![deny(dead_code, unreachable_code)]
  |         ^^^^^^^^^

error: aborting due to 2 previous errors


full stdout:


FAILURES:
    tests/actual_tests_bless/aux_build_not_found.rs
    tests/actual_tests_bless/aux_proc_macro_misuse.rs
    tests/actual_tests_bless/aux_proc_macro_no_main.rs
    tests/actual_tests_bless/compile_flags_quotes.rs
    tests/actual_tests_bless/compiletest-rs-command.rs
    tests/actual_tests_bless/failing_executable.rs
    tests/actual_tests_bless/no_main.rs
    tests/actual_tests_bless/no_main_manual.rs
    tests/actual_tests_bless/no_test.rs
    tests/actual_tests_bless/non_top_level_configs.rs
    tests/actual_tests_bless/pass_with_annotation.rs
    tests/actual_tests_bless/revised_revision.rs
    tests/actual_tests_bless/revisioned_executable.rs (revision panic)
    tests/actual_tests_bless/revisioned_executable_panic.rs (revision run)
    tests/actual_tests_bless/revisions_bad.rs (revision bar)
    tests/actual_tests_bless/rustfix-fail-revisions.rs (revision a)
    tests/actual_tests_bless/rustfix-fail-revisions.rs (revision b)
    tests/actual_tests_bless/rustfix-fail.rs
    tests/actual_tests_bless/unknown_revision.rs
    tests/actual_tests_bless/unknown_revision2.rs
    tests/actual_tests_bless/wrong_diagnostic_code.rs

test result: FAIL. 21 failed; 14 passed; 3 ignored;

Building dependencies ... ok

FAILED TEST: tests/actual_tests_bless_yolo/revisions_bad.rs (revision `bar`)
command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/tests/integrations/basic-fail/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests_bless_yolo/revisions_bad.rs" "--cfg=bar" "--edition" "2021"

error: ``main` function not found in crate `revisions_bad`` not found in diagnostics outside the testfile
 --> tests/actual_tests_bless_yolo/revisions_bad.rs:4:31
  |
4 | //@[bar] error-in-other-file: `main` function not found in crate `revisions_bad`
  |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected because of this pattern
  |

full stderr:
error[E0601]: `main` function not found in crate `revisions_bad`
  --> tests/actual_tests_bless_yolo/revisions_bad.rs:10:2
   |
10 | }
   |  ^ consider adding a `main` function to `tests/actual_tests_bless_yolo/revisions_bad.rs`

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0601`.

full stdout:


FAILURES:
    tests/actual_tests_bless_yolo/revisions_bad.rs (revision bar)

test result: FAIL. 1 failed; 2 passed;

Building dependencies ... ok

FAILED TEST: tests/actual_tests/bad_pattern.rs
command: "$CMD" "tests/actual_tests/bad_pattern.rs" "--edition" "2021"

full stderr:
No such file or directory
full stdout:
could not spawn `"invalid_foobarlaksdfalsdfj"` as a process


FAILED TEST: tests/actual_tests/executable.rs
command: "$CMD" "tests/actual_tests/executable.rs" "--edition" "2021"

full stderr:
No such file or directory
full stdout:
could not spawn `"invalid_foobarlaksdfalsdfj"` as a process


FAILED TEST: tests/actual_tests/executable_compile_err.rs
command: "$CMD" "tests/actual_tests/executable_compile_err.rs" "--edition" "2021"

full stderr:
No such file or directory
full stdout:
could not spawn `"invalid_foobarlaksdfalsdfj"` as a process


FAILED TEST: tests/actual_tests/exit_code_fail.rs
command: "$CMD" "tests/actual_tests/exit_code_fail.rs" "--edition" "2021"

full stderr:
No such file or directory
full stdout:
could not spawn `"invalid_foobarlaksdfalsdfj"` as a process


FAILED TEST: tests/actual_tests/filters.rs
command: "parse comments"

error: `x86_64` is not a valid condition, expected `on-host`, /[0-9]+bit/, /host-.*/, or /target-.*/
 --> tests/actual_tests/filters.rs:1:9
  |
1 | //@only-x86_64
  |         ^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests/foomp.rs
command: "$CMD" "tests/actual_tests/foomp.rs" "--edition" "2021"

full stderr:
No such file or directory
full stdout:
could not spawn `"invalid_foobarlaksdfalsdfj"` as a process


FAILED TEST: tests/actual_tests/foomp2.rs
command: "$CMD" "tests/actual_tests/foomp2.rs" "--edition" "2021"

full stderr:
No such file or directory
full stdout:
could not spawn `"invalid_foobarlaksdfalsdfj"` as a process


FAILED TEST: tests/actual_tests/pattern_too_many_arrow.rs
command: "parse comments"

error: //~^ pattern is trying to refer to 7 lines above, but there are only 2 lines above
 --> tests/actual_tests/pattern_too_many_arrow.rs:3:8
  |
3 |     //~^^^^^^^ ERROR: mismatched types
  |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests/pattern_too_many_arrow_above.rs
command: "parse comments"

error: //~v pattern is trying to refer to line 8, but the file only has 6 lines
 --> tests/actual_tests/pattern_too_many_arrow_above.rs:2:22
  |
2 |     //~vvvvvv ERROR: mismatched types
  |                      ^^^^^^^^^^^^^^^^
  |

full stderr:

full stdout:



FAILED TEST: tests/actual_tests/rustc_ice.rs
command: "$CMD" "tests/actual_tests/rustc_ice.rs" "-Ztreat-err-as-bug" "--edition" "2021"

full stderr:
No such file or directory
full stdout:
could not spawn `"invalid_foobarlaksdfalsdfj"` as a process

FAILURES:
    tests/actual_tests/bad_pattern.rs
    tests/actual_tests/executable.rs
    tests/actual_tests/executable_compile_err.rs
    tests/actual_tests/exit_code_fail.rs
    tests/actual_tests/filters.rs
    tests/actual_tests/foomp.rs
    tests/actual_tests/foomp2.rs
    tests/actual_tests/pattern_too_many_arrow.rs
    tests/actual_tests/pattern_too_many_arrow_above.rs
    tests/actual_tests/rustc_ice.rs

test result: FAIL. 10 failed;


running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 

